src/vm/runners/builtin_runner/bitwise.rs

```diff
@@ -96,6 +97,10 @@ impl BuiltinRunner for BitwiseBuiltinRunner {
     fn as_any(&self) -> &dyn Any {
         self
     }
+
+    fn get_memory_accesses(&self, _runner: &CairoRunner) -> std::collections::HashSet<Relocatable> {
```

Comment:
Please implement this method, (In a separate PR, and add that PR as a dependency to this one), as we can't have todo!() in the main branch

Reply:
[Done](https://github.com/lambdaclass/cairo-rs/pull/508).